scripts/CinnabarLabTradeRoom.asm

```diff
@@ -7,17 +7,17 @@ CinnabarLabTradeRoom_TextPointers:
 	dw Lab2Text3
 
 Lab2Text1:
-	TX_FAR _Lab2Text1
-	db "@"
+	text_far _Lab2Text1
+	text_end
 
 Lab2Text2:
-	TX_ASM
+	text_asm
 	ld a, $7
 	ld [wWhichTrade], a
 	jr Lab2DoTrade
 
 Lab2Text3:
-	TX_ASM
+	text_asm
 	ld a, $8
```

Comment:
We could make constants at some point for these scripted trade indices.

Reply:
Definitely. "Make constants for the various hard-coded magic numbers" is a general long-term todo, along with "identify the unknown_asm_xxxx labels".